lib/ohai/plugins/rackspace.rb

```diff
@@ -43,13 +43,27 @@ def has_rackspace_metadata?
     false
   end
 
+  # Checks for the rackspace manufacturer on Windows
+  # === Return
+  # true:: If the rackspace cloud can be identified
+  # false:: Otherwise
+  def has_rackspace_manufacturer?
+    return false unless RUBY_PLATFORM =~ /mswin|mingw32|windows/
+    require "wmi-lite/wmi"
+    wmi = WmiLite::Wmi.new
+    if wmi.first_of("Win32_ComputerSystem")["PrimaryOwnerName"] == "Rackspace"
+      Ohai::Log.debug("Plugin Rackspace: has_rackspace_manufacturer? == true")
+      return true
+    end
```

Comment:
Should have a final false for good measure :)

Reply:
Nil evals perfectly here and this is the same pattern we're using in all our other methods